packages/enet.rb

```diff
@@ -0,0 +1,23 @@
+require 'package'
+
+class Enet < Package
+  description 'ENet reliable UDP networking library'
+  homepage 'https://github.com/lsalzman/enet'
+  git_hashtag '4f8e9bdc4ce6d1f61a6274b0e557065a38190952'
+  @_ver = git_hashtag[0, 7]
+  version @_ver
+  license 'GPL-2'
+  compatibility 'all'
+  source_url 'https://github.com/lsalzman/enet.git'
+  
+
+  def self.build
+    system 'autoreconf -vfi'
```

Comment:
Is this line absolutely necessary?

Reply:
Yes-- the source does not come with `configure` scripts and the like. See the readme of the build.